src/request.coffee

```diff
@@ -3,10 +3,24 @@ PlatformAPI = require 'pinejs-client'
 Promise = require 'bluebird'
 request = require 'request'
 url = require 'url'
+osRelease = require './lib/os-release'
+
+osVersion = osRelease.getOSVersionSync(config.hostOsVersionPath)
+osVariant = osRelease.getOSVariantSync(config.hostOsVersionPath)
+supervisorVersion = require('./lib/supervisor-version').replace('v', '')
```

Comment:
I don't know what values `process.env.SUPERVISOR_TAG_EXTRA` can have, but if it contains `v`s they'll be removed. Maybe use this regex instead `/^v/`?